test/english.lisp

```diff
@@ -1,44 +1,29 @@
 (cl:in-package #:spell.test)
 
-(fiveam:def-suite* :spell.compare
+(fiveam:def-suite* :spell.english
   :in :spell)
 
-(defun map-dictionary-entries (function)
-  (with-open-file (stream (asdf:system-relative-pathname
-                           "spell/test" "data/english.txt"))
-    (spell::map-dictionary-file-entries function stream)))
+(test english.non-existing-word
+  "Test dictionary lookup with strings that are not existing words."
+  (is (null (spell:english-lookup "no-such-word")))
+  ;; A prefix of an existing word exercises a different code path.
+  (is (null (spell:english-lookup "fuc"))))
 
-(defun progress-reporter (&key (stream *trace-output*) (indicator #\.))
-  (let ((i 0))
-    (lambda ()
-      (multiple-value-bind (j k) (floor i 1000)
-        (when (zerop k)
-          (when (and *print-right-margin*
-                     (zerop (mod j *print-right-margin*)))
-            (format stream "~&; "))
-          (write-char indicator stream)
-          (force-output stream)))
-      (incf i))))
-
-(test compare.english
+(test english.compare
```

Comment:
Would it make sense to add a variant of these tests for `spell/simple`, so that this system is exercised as well?

Reply:
The only way of testing the simple system that I can think of is via a separate SBCL invocation in the GitHub Actions workflow since `spell` and `spell/simple` are mutually exclusive within a single image.

However, if we turn `spell/simple` into an alias for `spell`, there is little need to test the two separately.

Reply:
Yep, the idea merging the two makes this comment moot.